tests/sentry/backup/test_models.py

```diff
@@ -61,3 +62,19 @@ def test_organization(self):
         user = self.create_user()
         self.create_organization(owner=user)
         self.import_export_then_validate()
+
+    def test_actor(self):
+        self.create_user(email="test@example.com")
+        self.create_team(name="pre save team", organization=self.organization)
+        self.import_export_then_validate()
```

Comment:
Once you pull in the PR mentioned above, all of the `self.import_export_then_validate()` lines will need to be returned.